src/utils/url.js

```diff
@@ -30,3 +29,8 @@ export function httpPrefixed(url) {
 export function isValidUrl(url) {
   return url.match(urlRegex)
 }
+
+export function standardizeUrlForPageSettings(url) {
```

Comment:
There is a nice npm package for url operations like this - https://www.npmjs.com/package/url-parse (and some of the functions above)

Reply:
Thanks, it's good to know.